### PR TITLE
pulled changes from encryption

### PR DESCRIPTION
**Pull request by WhitePolaris**

### CODE REVIEW

File: PyGame_Project/MVC/Model/encryption.py [hunk 1]
```diff
@@ -18,20 +18,22 @@ def encrypt(plain_text, password):
     cipher_config = AES.new(private_key, AES.MODE_GCM)
 
     # return a dictionary with the encrypted text
```
Comment (EthanWright24) [inline PyGame_Project/MVC/Model/encryption.py:20]: returns a tuple not dict

File: PyGame_Project/MVC/Model/encryption.py [hunk 2]
```diff
@@ -18,20 +18,22 @@ def encrypt(plain_text, password):
...
-    cipher_text, tag = cipher_config.encrypt_and_digest(bytes(plain_text, 'utf-8'))
-    return {
-        'cipher_text': b64encode(cipher_text).decode('utf-8'),
```
Comment (EthanWright24) [inline PyGame_Project/MVC/Model/encryption.py]: encode then decode?

File: PyGame_Project/MVC/Model/encryption.py [hunk 3]
```diff
@@ -18,20 +18,22 @@ def encrypt(plain_text, password):
...
+    return str(dict).replace('\'' , "\"")
 
 def decrypt(enc_dict, password):
     # decode the dictionary entries from base64
-    salt = b64decode(enc_dict['salt'])
-    cipher_text = b64decode(enc_dict['cipher_text'])
-    nonce = b64decode(enc_dict['nonce'])
-    tag = b64decode(enc_dict['tag'])
+    enc_dict = json.loads(enc_dict)
+    salt = b64decode(enc_dict["salt"])
```
Comment (EthanWright24) [inline PyGame_Project/MVC/Model/encryption.py:33]: why encode in the decrypted function?

**Review by EthanWright24 — DISMISSED** [review]
- import for test_encryption is wrong

File: PyGame_Project/MVC/Model/encryption.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
 
-import hashlib#, pycryptodome
```
[the remark this answers is not in the document]
Comment (WhitePolaris) [reply PyGame_Project/MVC/Model/encryption.py]: Crypto lib used in encryption.py utilizes the library

File: PyGame_Project/MVC/Model/encryption.py [hunk 5]
```diff
@@ -8,7 +8,7 @@
 ## -------- encrypt / decrypt ----------- ##
 def encrypt(plain_text, password):
```
[the remark this answers is not in the document]
Comment (WhitePolaris) [reply PyGame_Project/MVC/Model/encryption.py:9]: changed to wordList

**Review by EthanWright24 — APPROVED** [review]

**Pull request merged**